testing/sdk_diag.py

```diff
@@ -162,6 +167,15 @@ def _dump_plans(item: pytest.Item, service_name: str):
             f.write('\n') # ... and a trailing newline
 
 
+def _dump_threads(item: pytest.Item, service_name: str):
+    threads = sdk_cmd.service_request('GET', 'v1/debug/threads')
+    out_path = _setup_artifact_path(item, 'threads_{}.json'.format(service_name.replace('/', '_')))
```

Comment:
The threads endpoint doesn't output JSON though, correct? The filename seems to indicate this.

Also, would replacing / with __ not be more consistent?

Reply:
Good catch on the first point. I'm claiming "copying prior art" on the second one :).